packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -23,7 +23,7 @@
     },
     "tsConfig": {
       "type": "string",
-      "description": "The full path for the TypeScript configuration file, relative to the current workspace."
+      "description": "The full path for the TypeScript configuration file, relative to the current workspace. Applies to both libraries and applications."
```

Comment:
As discussed on slack, these descriptions here are for the application specific builder. 

This builder is not able to build a library.